pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -502,7 +502,15 @@ message GlobalConfig {
 
   google.protobuf.BoolValue operatorManageWebhooks = 41;
 
-  // The next available key is 44
+  // Settings for remote cluster.
+  google.protobuf.BoolValue istioRemote = 44;
+  google.protobuf.BoolValue createRemoteSvcEndpoints = 45;
+  google.protobuf.BoolValue remotePilotCreateSvcEndpoint = 46;
```

Comment:
// If set, a selector-less service and endpoint for istio-pilot are created with the remotePilotAddress IP,
  // which ensures the istio-pilot.<namespace> is DNS resolvable in the remote cluster.
  google.protobuf.BoolValue remotePilotCreateSvcEndpoint = 46;